en/ros/external_position_estimation.md

```diff
@@ -1,114 +1,166 @@
+[EKF2_EV_POS_X](../advanced/parameter_reference.md#EKF2_EV_POS_X), [EKF2_EV_POS_Y](../advanced/parameter_reference.md#EKF2_EV_POS_Y), [EKF2_EV_POS_Z](../advanced/parameter_reference.md#EKF2_EV_POS_Z) | Set the position of the vision sensor (or MoCap markers) with respect to the robot's body frame.
+
+> **Tip** Reboot the flight controller in order for parameter changes to take effect.
+
+Now, you will need to feed the external position data to the flight controller using the [VISION_POSITION_ESTIMATE](https://mavlink.io/en/messages/common.html#VISION_POSITION_ESTIMATE). 
+MAVROS provides an easy interface to do this through the `vision_pose_estimate` plugin.
+
 
-* External position estimate can be enabled by setting the `EKF_AID_MASK` to enable vision position and yaw fusion
-* To use the external height estimate for altitude estimation, set `EKF2_HGT_MODE` to use vision
-* Adjust the `EKF2_EV_DELAY` parameter. This value actually represents how far off the timestamp of the measurement is off from the "actual" time it was captured at. It can technically be set to 0 if there is correct timestamping (not just arrival time) and timesync (e.g NTP) between mocap and ROS computers. In reality, this needs some empirical tuning since delays in the entire Mocap->PX4 chain are very setup specific and there is rarely a well setup system with an entirely synchronised chain.
+#### Tuning EKF2_EV_DELAY {#tuning-EKF2_EV_DELAY}
 
-* Use `EKF2_EV_POS_X`, `EKF2_EV_POS_Y`, `EKF2_EV_POS_Z` to set the position of the vision sensor (or mocap markers) with respect to the robot's body frame.
-* Reboot the flight controller in order for the parameters to take effect.
+[EKF2_EV_DELAY](../advanced/parameter_reference.md#EKF2_EV_DELAY) is the *Vision Position Estimator delay relative to IMU measurements*.
 
-Now, you will need to feed the external position data to the flight controller using the [VISION_POSITION_ESTIMATE](https://mavlink.io/en/messages/common.html#VISION_POSITION_ESTIMATE). MAVROS provides an easy interface to do this through the `vision_pose_estimate` plugin.
+Or in other words, it is the difference between the vision system timestamp and the "actual" capture time that would have been recorded by the IMU clock (the "base clock" for EKF2).
 
+Technically this can be set to 0 if there is correct timestamping (not just arrival time) and timesync (e.g NTP) between MoCap and (for example) ROS computers. 
+In reality, this needs some empirical tuning since delays in the entire MoCap->PX4 chain are very setup-specific. 
+It is rare that a system is setup with an entirely synchronised chain!
 
-## LPE Tuning for Vision or Mocap
+A rough estimate of the delay can be obtained from logs by checking the offset between IMU rates and the EV rates:
 
-### Enabling external pose input
+![ekf2_ev_delay log](../../assets/ekf2/ekf2_ev_delay_tuning.png)
```

Comment:
@ChristophTobler Perhaps a dumb question, but how do you get this log graph as discussed in https://github.com/PX4/Devguide/pull/573#discussion_r238493427

Reply:
@ChristophTobler  Ping?

Reply:
FlightPlot. E.g Plot external data and the onboard estimate.

Reply:
Thanks @mhkabir - I'll add a note.